tools/acrn-crashlog/README.rst

```diff
@@ -0,0 +1,124 @@
+
+Building the ACRN-Crashlog
+**************************
+
+Build dependencies
+==================
+
+The ACRN-Crashlog tool depends on the following libraries (build and runtime):
+
+- libevent
+
+- OpenSSL
+
+- libxml2
+
+- systemd
+
+- telemetrics-client-dev (optional, detected at build time)
+
+For more detailed instructions on how to set-up your build environment, please refer to the :ref:`getting_started`.
+
+Build
+=====
+
+To build the ACRN-Crashlog
+
+.. code-block:: console
+
+   make
+
+To Clean the build:
+
+.. code-block:: console
+
+   make clean
+
+Installing the ACRN-Crashlog
+****************************
+
+Install dependencies
+====================
+
+Since ACRN-Crashlog runs as a probe of `telemetrics-client`_, it needs `telemetrics-client`_ (>=1.6.0) to be available on the system.
+
+Install
+=======
+
+To install the build
+
+.. code-block:: console
+
+   sudo make install
+
+Usage
+*****
+
+ACRN-Crashlog works as a probe of `telemetrics-client`_. It is autostarted, running as daemon and sending the crashlog path to `telemetrics-client`_.
+`telemetrics-client`_ records events of interest and reports them to the backend using telemd telemetrics daemon. The type of events/logs is implemented as probe. The work flow of acrnprobe and `telemetrics-client`_ is:
+
+.. code-block:: console
+
+            crashlog path                   log content
+   acrnprobe------------->telemetircs-client----------->backend
+
+For more details of telemetrics, please refer the `telemetrics-client`_ and `telemetrics-backend`_ website.
+
+Crashlog can be retrieved with telem_journal command:
+
+.. code-block:: console
+
+   telem_journal -i
+
+ACRN-Crashlog also provides a tool debugger to dump the specific process information:
+
+.. code-block:: console
+
+   debugger <pid>
+
+.. note::
+
+   You need to be ``root`` to use the ``debugger``.
+
+Source Code
+***********
+
+The source code structure:
+
+.. code-block:: console
+
+   acrn-crashlog/
+   ├── acrnprobe
+   │   └── include
+   ├── common
+   │   └── include
+   ├── data
+   └── usercrash
+       └── include
+
+- acrnprobe: to probe our tool on `telemetrics-client`_, and gathers all the crash/event logs on the platform. For the logs on hypervisor, it's collected with acrnlog. For the log on SOS, the userspace crash log is collected with usercrash, and the kernel crash log is collected with the inherent mechanism like ipanic, pstore and etc. For the log on AaaG, it's collected with monitoring the change of related folders on the sos image, like /data/logs/. acrnprobe also provides a flexible way to allow users to config which crash/event they want to collect through xml file easily.
```

Comment:
Can you enclose the tools and paths (e.g. acrnprobe, acrnlog, usercrash, ipanic, pstore, /data/logs, ... in double-backtics?

(can you do this for the whole bullet list?)